enclave-service/Dockerfile

```
@@ -0,0 +1,110 @@
+    iptables \
+    jq \
+    libnbd-bin \
+    nbd-client \
+    net-tools \
+    openssh-server \
+    socat \
+    && update-ca-certificates \
+    && rm -rf /var/cache/apt/archives /var/lib/apt/lists/*
+
+# Install AWS CLI v2
+RUN apt-get update && apt-get install -y --no-install-recommends --no-install-suggests \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)